src/esn.calendar.libs/app/components/event-preview-popover/event-preview-popover.component.js

```diff
@@ -0,0 +1,13 @@
+require('./event-preview-popover.controller');
```

Comment:
does this work without problems when trying `serve:prod`?

Reply:
Yes, of course.